tests/contracts-rust.spec.ts

```diff
@@ -141,8 +142,7 @@ describe("Rust Smart Contracts", () => {
     // 6. restores the contract
     // 7. checks that the restored contract is equivalent to the evicted.
 
-    const STORAGE_KEY =
-      "0xf40ceaf86e5776923332b8d8fd3bef849cadb19c6996bc272af1f648d9566a4c";
+    const STORAGE_KEY = (new Uint8Array(32)).fill(1);;
```

Comment:
TBH, I find hex encoded keys better: they are more readable and also they are grepable. 

Can we still use some utility to convert from a hex string to `Uint8Array`?

Reply:
I also think it looks nicer, but the storage key is  just a blake2b encoded Uint8Array, so there's no way to decode it back to the original Uint8Array.

I asked the same question 2 days ago and that was your answer to me. :)

Reply:
or do you mean `0x0101010101010101010101010101010101010101010101010101010101010101` instead of `(new Uint8Array(32)).fill(1);` and passing that to the `getContractStorage()` function?

Reply:
The only thing that I mean here is intead of using:

```
const STORAGE_KEY = (new Uint8Array(32)).fill(1);
```

to use

```
const STORAGE_KEY = decode_hex("0x0101010101010101010101010101010101010101010101010101010101010101");
```

if `decode_hex` has the return type of `Uint8Array` then the change will be striclty local (i.e only involing this line of code), and the value of `STORAGE_KEY`. In other words, this change is strictly about readability.

Your comment regarding that this is 1-to-1 mapping to Rust version makes sense, although I still prefer the hex version.

Reply:
Hmm, fine if we'd do that in all places. But we're generating the values programmatically in Rust and AssemblyScript so I honestly don't see the point in doing it differently in the tests.

And besides, `0xf40ceaf86e5776923332b8d8fd3bef849cadb19c6996bc272af1f648d9566a4c` was the actual storage key in subastrate, `0x0101010101010101010101010101010101010101010101010101010101010101` is just the hex encoded Uint8Array, converted to hex by my brain. 

In the ERC20 example I'm using `randomAsU8a(32)` to generate the accountId which is used as the`STORAGE_KEY`, there's no way to pre-calculate that as a hex string. So I'd rather keep it consistent throughout the tests

Reply:
I am not proposing to change all usages to hex. I am only proposing to describe **literals** as that and for the sake of consistency and only there where it makes sense. For now it is only:

```
(new Uint8Array(32)).fill(1);
```

nothing more.

In future, there might be other cases though. Imagine when you want to write other literals, e.g. consider this:

```
new Uint8Array([0x01, 0x02, 0x03, 0x04, ..., 0x1F]));
```

and compare it with a hex literal

```
decode_hex("01020304..1F");
```

At least for me, the hex literal option wins by a huge margin.

> And besides, 0xf40ceaf86e5776923332b8d8fd3bef849cadb19c6996bc272af1f648d9566a4c was the actual storage key in subastrate, 0x0101010101010101010101010101010101010101010101010101010101010101 is just the hex encoded Uint8Array, converted to hex by my brain.

I am not sure how is this related to our discussion here... Again, I am only speaking about local / cosmetic changes. I don't propose to change anything anywhere else, i.e. `getContractStorage`. Therefore, this should be irrelevant.

That said, this is so minuscule detail that already generated way too much discussion that it deserved : ) So feel free to leave it this way